app/views/admin/users/_current_roles.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="crayons-card p-6 grid gap-2">
+  <h2>Current Roles</h2>
+  <p>(View full history in notes below)</p>
+  <div class="crayons-card__body">
+    <ul>
+      <% @user.roles.each do |role| %>
+        <% if role.name == "banned" %>
```

Comment:
**suggestion (non-blocking):** We have some similar code in `app/views/admin/users/_activity.html.erb`. Maybe we could use a helper like `role_display_name` here? Something like

```ruby
role_display_name(role)
  return "suspended" if role.name == "banned"

  role.name
end
```

Then you cold change the code here to this:

```ruby
<% if role.name == "banned" || role.name == "super_admin" || current_user.id == @user.id %>
  <li><%= role.name.titlecase %></li>
```

But that's probably better done in a follow-up PR.

Reply:
Sounds good to me! Thanks so much for the suggestion @citizen428 😊

Reply:
Circled back around to this in another PR - done! ✅